open_samus_returns_rando/specific_patches/game_patches.py

```diff
@@ -112,3 +112,8 @@ def _patch_reverse_area8(editor: PatcherEditor, configuration: dict):
         editor.remove_entity(
             {"scenario": "s100_area10", "layer": 9, "actor": "LE_ValveQueen"}
         )
+        # This spawnpoint is unused and will not be used for random start
+        # There might be a more efficient way of disabling the elevator but this works just fine
+        editor.remove_entity(
+            {"scenario": "s110_surfaceb", "layer": 5, "actor": "StartPoint0"}
+        )
```

Comment:
A  boolean for what? I wanted to check for the wall being removed directly, but since it's in a different scenario it doesn't exist. This was the next best thing I could think of.

Reply:
From what I understood, you remove the startpoint if the patch is present and otherwise keep it. It is just a boolean for "patch is active or not" a.k.a passing `reverse_area8` from the patcher to the lua like e.g. the `reveal_map_on_start` boolean

Reply:
I couldn't figure out how to have the patcher interact with the lua in a more direct way. If there is a way then I'd rather do that.